server/util/json.go

```diff
@@ -29,14 +29,14 @@ func GetFromUrl(url string) ([]byte, error) {
 	return ret, err
 }
 
-func ParseJson[D string | []byte, T interface{}](data D, target *T) error {
+func ParseJson[D string | []byte, T any](data D, target *T) error {
 	if err := json.Unmarshal([]byte(data), target); err != nil {
 		return err
 	}
 	return nil
 }
```

Comment:
This function does not do anything, right?

It feels like in the code, you would have to use ParseJson in the exact same way as unmarshal.